packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -97,6 +97,35 @@ When using red text for destructive actions, it is still recommended to require
 
 <img src="./img/red-delete-text.png" alt="drilldown menu" width="935"/>
 
+### Disabling menus and menu options
+Menus or menu items can be disabled for multiple reasons. The reason an action is disabled informs which design pattern you should use.
+
+1. **An action can be performed, but is not recommended.**
+When an action can be performed but may result in an undesirable outcome, do not disable it. 
+
+Instead, add a [confirmation model](components/model). When the user clicks on the action, use a modal to explain the potential consequences and ask the user if they are sure they want to proceed.
+
+Example: A user wants to start maintenance on a master node.
+
+2. **The user doesn’t have the correct permissions to perform an action.**
+If a user doesn’t have permission to perform an action, disable the dropdown options.
+
+Example: A user with view-only privileges looks at a Pod’s details page and the options within the Actions dropdown are disabled.
+
+3. **An action is unavailable due to an unmet prerequisite, condition, or status.** 
+If a user needs to complete a prerequisite to enable an action, disable the action and add a [tooltip](components/tooltip). The tooltip should explain what the user needs to do to enable the action. 
+
+Example: A user can’t perform bulk actions until they select resources in the list.
+
+If an action is dependent on an unmet condition or status in the console, disable the action with a tooltip explaining why the action currently is unavailable.
+
+Example: A user cannot view past logs until their container is finished restarting.
+
+4. **An action cannot be performed due to a product constraint or rule.**
+If an action cannot be taken because of a product constraint or rule, hide the action. 
+
+Example: A user can’t delete a Red Hat template or remove an Environment variable from a managed resource.
```

Comment:
This one is also confusing to me - why wouldn't a user be able to delete something, or remove something - wouldn't that also be permission related? What would be an example of a product constraint. I would make the examples more descriptive so that they're a bit more applicable/clear